o/opentracing/redhat_ubi8/build.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+# Package       : opentracing
+# Version       : 1.6.0
+# Source repo   : https://github.com/opentracing/opentracing-cpp
+# Tested on     : RHEL_8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora
+#
+# Disclaimer: This script has been tested in root mode on given platform using
+#             the mentioned version of the package. It may not work as expected
+#             with newer versions of the package and/or distribution.
+#             In such case, please contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+# clone branch/release passed as argument, if none, use master
+
+if [ -z $1 ] || [ "$1" == "lasttestedrelease" ]
+then
+       # As on 19 Sep 2019, release 1.6.0was latest
+       BRANCH="--branch v1.6.0"
+else
+       BRANCH="--branch $1"
+fi
```

Comment:
Few minor comments, rest lgtm.
Looks like it doesn't build master by default (i.e. if no branch is specified by user)?
Build specific branch if supplied and advisable to avoid any hard-coding if possible.

Below altered snippet will ensure master is built by default or specific branch if supplied
```
if [ -z $1 ]
then
       BRANCH=""
else
       BRANCH="--branch $1"
fi
```
If user merely wishes to build/test off master, I assume this should be run: "./build.sh runtests".
Hopefully, the script handles this.

If this is too much work, we can merge this and make an enhancement later, up to you.